js-core/homeworks/homework-13/src/main.js

```diff
@@ -0,0 +1,59 @@
+
+/*
+ * Вы должны создать имитацию медленной базы данных.
+ * TASK - 1 Сделайте Класс Database с методами
+ *
+ *  query
+ *
+ *  При запуске метода query запустите внутренний таймаут, который будет длиться 5 секунд.
+ *  При поступлении еще 1 запроса(если вызвать метод еще раз),
+ *  таймаут должен стартануть сначала
+ *  и ответ должен прийти снова через 5 секунд
+ *
+ * */
+
+class DataBase {
+    constuctor () {}
```

Comment:
constuctor -> conductor -> constructor :)

It's not really necessary to create a constructor